susemanager/bin/mgr-setup

```diff
@@ -423,6 +423,26 @@ if [ -f $MANAGER_COMPLETE ]; then
 fi
 }
 
+
+
+#TODO most of the apache configuration are set in the perl script, so these changes should be there...but since we want to deprecate perl, the final
+# goal would be to move everything here.
+#
+setup_apache() {
+    sed -i 's|DocumentRoot "/srv|DocumentRoot "/usr/share/susemanager|g' /etc/apache2/default-server.conf
+    sed -i 's|<Directory "/srv|<Directory "/usr/share/susemanager|g' /etc/apache2/default-server.conf
+    sed -i 's|<Directory "/srv|<Directory "/usr/share/susemanager|g' /etc/apache2/conf.d/zz-spacewalk-www.conf
+    sed -i 's|ScriptAlias /cgi-bin/ "/srv|ScriptAlias /cgi-bin/ "/usr/share/susemanager|g' /etc/apache2/default-server.conf
+    sed -i 's|DocumentRoot "/srv|DocumentRoot "/usr/share/susemanager|g' /etc/apache2/vhosts.d/vhost-ssl.conf
+    sed -i 's|DocumentRoot "/srv|DocumentRoot "/usr/share/susemanager|g' /etc/apache2/vhosts.d/vhost-ssl.template
+}
+
+#TODO move this on the permission check function
+setup_permission() {
+    chown tomcat:tomcat -R /usr/share/susemanager/www/htdocs
+    chmod 777 -R /usr/share/susemanager/www/htdocs
+}
```

Comment:
I was wondering the same

Reply:
right, currently upgrade is not managed at all. I need to add somewhere this part

Reply:
Why do we need to change those permissions? Isn't it enough to set them on the folder in the spec file?

Reply:
It should, I just add that for keeping it easy. Once the tests will be ok, I'd like to remove `setup_permission` at all

Reply:
I think this commit should fix the concern https://github.com/uyuni-project/uyuni/pull/7651/commits/ef506192fc1cead42a0e52860c9fd2c954a1a29b

Instead of manipulating files, I created new ones (and disable the previous ones). In this way the upgrade should be ok. I'm now going to test if permission are ok. Please let me know what you think about this change